### ISSUE TITLE
Add configuration for language / timezone of diagram footer

### ISSUE THREAD

**Issue by ffe4** [structurizr/structurizr#150]
I assume this is the correct repo, but for context, I'm using the structurizr lite web interface using the docker container.

## Context

Each diagram has the title and last modification time at the bottom. Currently, the timezone and locale of the client machine is used to populate this field.

![gb](https://user-images.githubusercontent.com/61800298/220941980-9938faba-a395-4954-9baf-ea98323a47c4.PNG)

## Issue

This is not great for international collaboration, as it means that during presentations or in exports (e.g. for inclusion in other documentation systems) diagrams might contain metadata in a language the target audience is not able to understand.

![jp](https://user-images.githubusercontent.com/61800298/220941984-277d53bd-0349-434a-b82b-aeec4b8382d7.PNG)

On a side note, this also ignores the `ACCEPT-LANGUAGE` header sent by the browser.

## Suggestion

I would suggest two options on the workspace level, to configure timezone and locale respectively. This way a team can specify a common standard. I would not mind keeping the timezone local, as long as I can specify a language so that everyone understands it. However, in some contexts it could also be useful to take the guessing out of the date but just decide on a reference time zone.

As an addition, or an alternative, it could also be an option during export, to at least make that version accessible to your target audience.

**Comment by simonbrowndotje** [comment]
I can see how that could be confusing for people! I've added a way to configure the locale and timezone via a couple of viewset properties, for example:

```
workspace {

    model {
        user = person "User"
    }

    views {
        properties {
            structurizr.locale de-DE
            structurizr.timezone Australia/Melbourne
        }
        
        systemLandscape {
            include *
            autolayout
        }
    }
    
}
```

The cloud service and on-premises installation have some server-side rendered pages that include things like the last modified date, so I've transferred the issue to this repo. I would also like to document the above properties somewhere.

But the above is available in Lite 2986.

**Comment by ffe4** [comment]
Works like a charm, thank you for the quick fix!

> I would also like to document the above properties somewhere.

Regarding documentation, as I have spent quite a bit of time on the [DSL Reference](https://github.com/structurizr/dsl/blob/master/docs/language-reference.md) lately, I was gonna say just put it under [properties](https://github.com/structurizr/dsl/blob/master/docs/language-reference.md#properties). 

However, I also noticed that this part, despite being linked to from everywhere in the document, is placed in-between the model and views blocks, with a heading on the same markdown level as everything surrounding it, but isn't linked anywhere in the ToC.

That's a bit confusing, but also another issue, and I see that you're working on the docs right now. I've submitted a quick fix for the docs [#226](https://github.com/structurizr/dsl/pull/226) but feel free to reject. From my side this issue can otherwise be closed.
